Automattic/wp-calypso
# Add premium domain badge in the domain status screen

#### Changes proposed in this Pull Request

* We should mark premium domains as such in the domain status page, so here's a badge for premium domains in the domain status page
* The active colour should also change for premium domains to be the blue we use for the badge
* The item list should also be marked with the premium active color

see p99Zz8-1mx-p2

Here's how it looks when in domain list:
<img width="1079" alt="Screenshot 2020-09-23 at 16 21 30" src="https://user-images.githubusercontent.com/1355045/94018727-917eba00-fdb9-11ea-8c55-8a3d45aa580c.png">

And here is how it looks in the domain status screen:
<img width="750" alt="Screenshot 2020-09-23 at 16 21 52" src="https://user-images.githubusercontent.com/1355045/94018731-92afe700-fdb9-11ea-95dd-1e08bf6428a5.png">

#### Testing instructions

* Depends on D50025-code
* Open a premium domain in your domain management screen and verify that the "Premium domain" badge is visible.
* Verify the header says "Premium domain" and not "Registered domain"
* Verify that the domain status color for premium domain is blue
* Verify that in the domain list the status color is also blue
